DeepSpeech.ipynb

```diff
@@ -899,9 +899,9 @@
     " \n",
     "and retain the original ```labels``` (Y).\n",
     " \n",
-    "```decoded```, ```labels```, the optimization gradient, ```total_loss``` and ```avg_loss``` are collected into the respectful arrays ```tower_decodings, tower_labels, tower_gradients, tower_total_losses, tower_avg_losses``` (dimension 0 being the tower).\n",
+    "```decoded```, ```labels```, the optimization gradient and ```total_loss``` are collected into their respectful arrays ```tower_decodings, tower_labels, tower_gradients, tower_total_losses``` (dimension 0 being the tower).\n",
```

Comment:
still small nit here, s/respectful/respective/ (I realise the mistake was there before, but seeing as we're changing this line, we may as well fix it)